tasks/__init__.py

```diff
@@ -258,25 +265,15 @@ def requirements(ctx, base=False, addons=False, release=False, dev=False, all=Fa
         addon_requirements(ctx)
     # "release" takes precedence
     if release:
-        req_file = os.path.join(HERE, 'requirements', 'release.txt')
-        ctx.run(
-            pip_install(req_file),
-            echo=True
-        )
+        install_from_txt(ctx, os.path.join(HERE, 'requirements', 'release.txt'))
     else:
         if dev:  # then dev requirements
-            req_file = os.path.join(HERE, 'requirements', 'dev.txt')
-            ctx.run(
-                pip_install(req_file),
-                echo=True
-            )
+            install_from_txt(ctx, os.path.join(HERE, 'requirements', 'dev.txt'))
+        elif base:  # then base requirements
```

Comment:
I changed this to an `elif` because these `.txt`  all contain a `-r ../requirements.txt` so they incorporate those reqs anyway.